### PR TITLE
86764 - Signer info section - form 10-7959a

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR adds screens to the signer info chapter of form 10-7959a. It includes role, name, address, phone, and relationship screens.

- Adds new screens to signer info chapter
- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86764

## Testing done

- Manual

## Screenshots

|         | Screens |
| ------- | ------ |
| Role |Screenshot 2024-07-08 at 13 38 50|
| Name |Screenshot 2024-07-08 at 13 39 04|
| Address |Screenshot 2024-07-08 at 13 39 20|
| Contact |Screenshot 2024-07-08 at 13 39 31|
| Relationship |Screenshot 2024-07-08 at 13 40 11|

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-7959a/chapters/signerInformation.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959a/chapters/signerInformation.js:1]: This file holds all the new page schemas

File: src/applications/ivc-champva/10-7959a/chapters/signerInformation.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    type: 'object',
+    required: ['certifierPhone'],
+    properties: {
+      titleSchema,
+      certifierPhone: phoneSchema,
+    },
+  },
+};
+
+export const certifierRelationshipSchema = {
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959a/chapters/signerInformation.js:91]: Relationship schema has an optional text field that only appears if the "other" radio option is selected.

File: src/applications/ivc-champva/10-7959a/config/form.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959a/config/form.js:1]: Added the new schemas to five new page configs in the main form config

**Review by balexandr — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**